plugins/woocommerce/tests/e2e-pw/playwright.config.js

```diff
@@ -70,9 +70,15 @@ const config = {
 	},
 	projects: [
 		{
-			name: 'Chrome',
+			name: 'default',
 			use: { ...devices[ 'Desktop Chrome' ] },
 		},
+		{
+			name: 'Gutenberg',
+			use: { ...devices[ 'Desktop Chrome' ] },
+			testIgnore:
+				/.*smoke-tests\/*|.*js-file-monitor\/*|.*admin-tasks\/*|.*activate-and-setup\/*|.*admin-analytics\/*|.*admin-marketing\/*/,
+		},
```

Comment:
Nice use of projects here.  Think this'll be a great way to segment the different types of tests we need to run.